### ISSUE TITLE
Mesen-S-DevWin-0.3.0.10 X sound problems

### ISSUE THREAD

**Issue by AlanRibas** [SourMesen/Mesen-S#76]
I just downloaded the "Mesen-S-DevWin-0.3.0.10" version and noticed that the sound has suffered some errors. There is a kind of very discreet cut in the sound.
I tested this new version with a last version using "Mesen-S-DevWin-0.3.0.2". In this older version this problem of apparent minor cuts in sound does not exist.
It sure is a new bug.

**Comment by SourMesen** [comment]
There hasn't really been any sound-related change in a very long time, and I can't hear any issues on my end.  Does it happen in all games?  How fast (in terms of FPS) can your PC run the games if you try to increase the speed?  If you're barely hitting 60 FPS, that might be the problem.

**Comment by AlanRibas** [comment]
The game I was testing was just one. "Super Castlevania IV (USA)".
That day I noticed a very subtle cut sound (on the game's introduction screen). Something that really should pay attention to realize. But today I did tests with this game again and the two versions of the emulators got the same way.
My computer has no problem emulating "MESEN S"
Itel - i5 - 6600
Z170N Gaming Drives 5.
GTX 970

Sorry to make you waste your time on this useless post.
Next time I'll do more tests before reporting anything.
